example/glue/CollectorLanguageInterfaceImpl.cpp

```diff
@@ -197,16 +213,12 @@ MM_CollectorLanguageInterfaceImpl::parallelDispatcher_handleMasterThread(OMR_VMT
 void
 MM_CollectorLanguageInterfaceImpl::generationalWriteBarrierStore(OMR_VMThread *omrThread, omrobjectptr_t parentObject, fomrobject_t *parentSlot, omrobjectptr_t childObject)
 {
```

Comment:
Why is this implementation still in the Impl class?  Is not exactly the same for all languages?